R/R.r

```diff
@@ -1,6 +1,6 @@
 # R("-e 'str(as.list(Sys.getenv()))' --slave")
 R <- function(options, path = tempdir(), env_vars = NULL, ...) {
-  options <- paste("--vanilla", options)
+  options <- paste("--no-site-file", "--no-environ", options)
```

Comment:
Hm... `--vanilla` doesn't imply `--no-save`, but I agree it's better to add it anyway.